pages/storefront/edit.tsx

```diff
@@ -286,6 +288,22 @@ export default function Edit() {
               >
                 <Input.TextArea />
               </Form.Item>
+              <Form.Item
+                name={['meta', 'mint']}
```

Comment:
Hmm, should we let people change mints on a store? Might be better to have them make a new store, @kristianeboe thoughts?

Reply:
Good question. I think only new stores should be allowed to set SPL mint. However they might need to upload a custom icon akin to the SOL circles, and those should be able to be updated from the edit page.

After launching this to production I suggest we at Holaplex offer a grace period where we offer to help already existing stores to change their mints once.

Reply:
I guess the complication here is in the lookup for naming and pricing? 

Or is there business logic behind restricting an edit?